ui/src/test/LeftNav.test.js

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter as Router } from 'react-router-dom';
+import LeftNav from '../Components/LeftNav/LeftNav';
+
+describe('<LeftNav/> component', () => {
```

Comment:
Is it a good idea to call it **Left**Nav? We might move it to right perhaps in future. Something like `NavPanel` would be better I guess. What do you think?

Reply:
Made it `Navigation`